pandas/_libs/dtypes/generic.pyx

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# cython: profile=False
+# cython: linetrace=False
+# distutils: define_macros=CYTHON_TRACE=0
+# distutils: define_macros=CYTHON_TRACE_NOGIL=0
+"""define generic base classes for pandas objects"""
+
+
+def create_pandas_abc_type(name, attr, comp):
+    """
+    define abstract base classes to enable isinstance type checking on our
+    objects
+    """
+    comp = set(comp)
+
+    if len(comp) == 1:
+        comp = list(comp)[0]
+        @classmethod
```

Comment:
does this actually matter? it's just more code 
i would leave the original

also when doing changes like this don't change code

very hard to detect what u changed

Reply:
pls respond here